docs/download_files.md

```diff
@@ -0,0 +1,61 @@
+# Downloadable Files
+
+Users will be able to download the gene expression data, QC report, and associated metadata for all samples that have been processed.
+These files are delivered as a zip file.
+The folder structure within the zip file is determined by whether a user selected to download all the samples associated with a project or selected to download individual samples.  
+
+## Download folder structure for project downloads: 
+_add in illustration of file structure for project_ 
+
+## Download folder structure for individual sample downloads: 
+_add in illustration of file structure for sample downloads_ 
+
+Folders for each sample (indicated by the `SCPCS` prefix) will contain the files for all libraries corresponding to that sample. 
+
+Each library will include the following files: `<scpca_library_id>_unfiltered.rds`, `<scpca_library_id>_filtered.rds`, `<scpca_library_id>_qc.html`, and `<scpca_library_id>_metadata.json`. 
+Every download will also include a `libraries_metadata.csv` file containing metadata associated with each library included in the download.
+
+## Unfiltered Counts Matrix 
+
+The unfiltered counts matrix, `<scpca_library_id>_unfiltered.rds`, is delivered as a RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
+Within the `SingleCellExperiment` is the counts matrix, where the rows correspond to genes or features and the columns correspond to cell barcodes. 
+Here, all potential cell barcodes that are identified after running [Alevin-fry](processing_information.html/#alignment-and-quantification-using-alevin-fry) are included in the counts matrix. 
+Summary statistics for each cell and gene can be found in the `colData` and `rowData` of the `SingleCellExperiment` object.
+The `SingleCellExperiment` object also contains metadata about that particular library, including the versions of Salmon and Alevin-fry used for pre-processing, information about the index used for transcriptome alignment, and parameters used for Alevin-fry. 
+See also [Using the provided RDS files in R.](link to FAQ)
+
+## Filtered Counts Matrix
```

Comment:
I like the idea of putting these under one heading of `Gene expression data`. In terms of describing the colData and rowData, I wasn't sure how much detail we wanted to include here and am thinking that increasing the level of detail might belong in a section where we discuss addition of QC and metadata. I think #14 starts to address this, but we might want to add information about adding per cell and per feature QC in the processing section, rather than here?